src/main/kotlin/no/nav/syfo/service/Oppgaveservice.kt

```diff
@@ -40,7 +41,9 @@ suspend fun handleRegisterOppgaveRequest(
                 behandlingstema = if (produceTask.behandlingstema != "ANY") produceTask.behandlingstema else { null },
                 aktivDato = LocalDate.parse(produceTask.aktivDato, DateTimeFormatter.ISO_DATE),
                 fristFerdigstillelse = LocalDate.parse(produceTask.fristFerdigstillelse, DateTimeFormatter.ISO_DATE),
-                prioritet = produceTask.prioritet.name
+                prioritet = produceTask.prioritet.name,
+                tildeltEnhetsnr = if (produceTask.tildeltEnhetsnr.isNotEmpty()) { produceTask.tildeltEnhetsnr } else null,
```

Comment:
Hvis vi setter tildeltEnhetsnummer overstyrer vi logikken for å sette riktig enhet som finnes i Oppgave, og som kan endres uten at vi får beskjed. Jeg ser at vi ikke setter den nå, men dette kan bli en potensiell felle tror jeg 😅